SEC/Securities/SecuritiesIssuance.rdf

```diff
@@ -547,6 +615,14 @@
 		<skos:definition>indicates a period of time in which investors can commit to purchase shares (or units) to be issued</skos:definition>
 	</owl:ObjectProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-sec-sec-iss;isIssuedInForm">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-agr-ctr;isEvidencedBy"/>
```

Comment:
I don't get use of this super property, though isEvidencedBy is very poorly defined as "is attested by", there is no way the SecurityForm is evidence for anything

Reply:
The contract document, either virtual or physical, such as a bearer bond, provides evidence for ownership.  How that evidence is manifested, i.e., the form it takes, is what this property is intended to be used for.  So that's why I selected isEvidencedBy, but I understand, I think, what you are saying here.  I couldn't use takesForm from FND, which has a very specific meaning, so I've replaced it with isRealizedBy.